src/components/Tags/Tags.tsx

```diff
@@ -0,0 +1,22 @@
+import cn from 'bem-cn-lite';
+
+import {Tag, TagType} from '../Tag/Tag';
+
+import './Tags.scss';
+
+const b = cn('tags');
+
+interface TagsProps {
+    tags: string[];
+    tagsType?: TagType;
+    className?: string;
+}
+
+export const Tags = ({tags, tagsType, className = ''}: TagsProps) => {
+    return (
+        <div className={`${b()} ${className}`}>
```

Comment:
no need for the `className` default value, and the mix is applied as `b(null, className)` without strings concatenation